pkg/manifests/config.go

```diff
@@ -162,15 +162,9 @@ func (c *Config) applyDefaults() {
 	if c.PrometheusK8sConfig.Retention == "" {
 		c.PrometheusK8sConfig.Retention = "15d"
 	}
-	if c.PrometheusK8sConfig.Resources == nil {
-		c.PrometheusK8sConfig.Resources = &v1.ResourceRequirements{}
-	}
```

Comment:
why do we need to remove this?

Reply:
Because of https://github.com/openshift/cluster-monitoring-operator/blob/df72275f1f5cdb19bfbc86da922c2e366f63bf87/pkg/manifests/manifests.go#L723-L725

Reply:
nevermind, I was too blind to see that it nil-checks `if c.PrometheusK8sConfig.Resources == nil` :man_facepalming: